package.json

```diff
@@ -14,20 +14,37 @@
   },
   "license": "MIT",
   "dependencies": {
-    "hoek": "2.x.x"
-  },
-  "peerDependencies": {
-    "hapi": ">= 8.x.x < 10"
+    "autoprefixer-stylus": "0.8.0",
```

Comment:
These version formats seem inconsistent. I thought it was n.x.x for hapi/CL modules and n.n.x for others